### ISSUE TITLE
Request: Add redirects to search results

### ISSUE THREAD

**Issue by ARR8** [kiwix/kiwix-android-custom#27]
So that, e.g., one could type JFK in the Wikivoyage app rather than John F. Kennedy International Airport. Would be more convenient for on-the-go use.

**Comment by kelson42** [comment]
@ARR8 Wikivoyages redirects should be available in the search. Do you have a concrete example where a redirect is available on wikivoyage.org and not in the app?

**Comment by ARR8** [comment]
Yes, see above. I cannot type jfk or JFK to get to the page on the airport.

**Comment by kelson42** [comment]
@ARR8 "jfk" is not a redirect to article "John F. Kennedy International Airport", "JFK" is in Wikipedia. And it is true is does not appear!
